lib/basicinfo.js

```diff
@@ -0,0 +1,7 @@
+module.exports = function basicYou(info) {
+	//var fun = info.split("");
+	//var funtimes;
```

Comment:
for production code (or for code you turn in), you can make a new branch off your existing branch, and hten remove the commented-out code and console logs from that branch. That way, you have clean/turn-in-able code on one branch, and you can always switch branches to see your commented-out code and console logs.